contracts/VotingEscrow.vy

```diff
@@ -346,6 +351,7 @@ def _deposit_for(_addr: address, _value: uint256, unlock_time: uint256, locked_b
         assert ERC20(self.token).transferFrom(_addr, self, _value)
 
     log Deposit(_addr, _value, _locked.end, type, block.timestamp)
+    log Supply(supply_before, self.supply)
```

Comment:
You won't believe it, but logging `(supply_before, supply_before + _value)` will probably cost less. Yes, self.supply is doing an expensive read again